packages/components/icons/package.json

```diff
@@ -22,10 +22,17 @@
       "./index.ts",
       "./custom-icon/index.ts",
       "./inline-svg/index.ts",
-      "./leading-icon/index.ts"
+      "./leading-icon/index.ts",
+      "./generated/*.tsx"
```

Comment:
This glob tells preconstruct that each of the icons should have an entrypoint in the package.